server/src/test/scala/services/crunch/CrunchMinuteSpec.scala

```diff
@@ -48,4 +49,15 @@ class CrunchMinuteSpec extends CrunchTestLike {
       dupes === expected
     }
   }
+
+  "Equality" >> {
+    "Given two CrunchMinutes with the same values but different last updated timestamps " +
+      "When I ask if they're equal " +
+      "The answer should be yes" >> {
+      val cm1 = CrunchMinute("T1", Queues.EGate, 0L, 1, 2, 3, 4, Option(5), Option(6), Option(7), Option(8), None)
+      val cm2 = CrunchMinute("T1", Queues.EGate, 0L, 1, 2, 3, 4, Option(5), Option(6), Option(7), Option(8), Option(1L))
+
+      cm1.isEqual(cm2) === true
```

Comment:
check if this is used - maybe rename to unchanged.

Reply:
Yup not used - removed